requirements.txt

```diff
@@ -1,6 +1,6 @@
 #
-# This file is autogenerated by pip-compile with python 3.10
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.8
```

Comment:
This seems weird.... why has it downgraded to python 3.8?

Reply:
Did you manually run it from your env?

Reply:
See NOTE in the BUILD.bazel:

```
    # NOTE: the requirements.txt reports the system python's version, not
    # the one specified in MODULE.bazel.
    # https://rules-python.readthedocs.io/en/latest/pypi-dependencies.html#pip-rules
 ```